src/open_inwoner/pdc/managers.py

```diff
@@ -33,6 +37,49 @@ def visible_for_user(self, user: User):
             return self.filter(visible_for_authenticated=True)
         return self.filter(visible_for_anonymous=True)
 
+    def filter_for_user_with_zaken(self, user: User):
+        """
+        Returns the categories linked to ZaakTypen for which the user has Zaken.
+        """
+        if not getattr(user, "bsn", None):
+            return self
+
+        cases = fetch_cases(user.bsn)
+
+        months_since_last_zaak_per_zaaktype = {}
+        for case in cases:
+            resolve_zaak_type(case)
```

Comment:
@alextreme this approach does mean that we have to do a fetch to get the zaaktype for each zaak, which isn't very efficient. I'm not sure how we could avoid this without something like an `include` parameter.

Reply:
Discussed: simplify this by storing the zaaktype URL on zaaktypeconfig and use that instead of identificatie on category, to avoid having to resolve here